lib/index.js

```diff
@@ -13,7 +13,9 @@ module.exports = function(context) {
     var bodyStartLine = body[0].loc.start.line;
     var openBraceLine = context.getTokenBefore(body[0]).loc.start.line;
 
-    if (bodyStartLine - openBraceLine < 2) {
+    if (bodyStartLine !== openBraceLine &&
```

Comment:
I think you have to check the function's closing brace to verify that it's a one liner. For example, this should be a violation:

```javascript
function foo() { var bar = 0;
  return bar;
}
```

You might also want to validate that the body's length is 0 or 1.